tests/modeladmin/models.py

```diff
@@ -28,6 +28,39 @@ class Concert(models.Model):
     ), blank=True)
 
 
+@python_2_unicode_compatible
```

Comment:
I am not sure what is the best way to do it since the models are currently being used in two different tests. I made the other changes.